src/excmds.ts

```diff
@@ -4363,6 +4363,22 @@ export function rot13(n: number) {
         body.currentNode.textContent = rot13_helper(t, n)
     }
 }
+/**
+ * Perform text jumbling.
+ *
+ * Shuffles letters except for first and last in all words in text nodes in the current tab. Only characters in
+ * the ASCII range are considered.
```

Comment:
Should probably add a reference to a site that explains the phenomenon : )

Reply:
Added